src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -1642,6 +1643,57 @@ impl<'a> TypeChecker<'a> {
                 )
             }
 
+            ("last_query_id", args) => {
```

Comment:
Can add some comments to describe the rewrite results. Like other rewriteable functions.